config/simplesamlphp-users.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/*
+  Specify user info for https://hub.docker.com/r/kristophjunge/test-saml-idp/ in the same format
+  as our Seneca IdP will give it.  This is modeled on the original config here:
+  https://github.com/kristophjunge/docker-test-saml-idp/blob/master/config/simplesamlphp/authsources.php
+*/
+$config = array(
+
+    'admin' => array(
+        'core:AdminPassword',
+    ),
+
+    'example-userpass' => array(
+        'exampleauth:UserPass',
+        'user1:user1pass' => array(
+            'uid' => array('1'),
+            'eduPersonAffiliation' => array('group1'),
+            /*
+              NOTE: we add a bunch of claims that we expect to get back from Seneca,
+              and need to simulate here.
+            */
+            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress' => 'user1@example.com',
+            'email' => 'user1@example.com',
+            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/givenname' => 'Johannes',
+            'http://schemas.xmlsoap.org/ws/2005/05/identity/claims/surname' => 'Kepler',
+            'http://schemas.microsoft.com/identity/claims/displayname' => 'Johannes Kepler'
```

Comment:
We better add a few more claims, since we will need a `username`:

```
  "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name": "username@seneca-domain.ca",
  "sAMAccountName": "username"
```

Give all 3 users unique usernames that match their email address.

Since you're creating this setup, you also get to name the users.  If you want to immortalize yourself, now is your chance!

Reply:
Should the email and name attributes have the same values? Our Seneca e-mail account should be used for both right?

Reply:
Also perhaps I should ask, cause I am not clear about it. What is sAMAccountName? It seems as though from my searching it is an older username that is of the form domain\username as opposed to the email-like user@domain.com that we use now. Do we all have these? Are these created by default when an account is made using the prefix of the user@domain.com users?